src/birthder.js

```diff
@@ -68,6 +68,41 @@
     }
   }
 
+  /** 
+   * Creating a room for all users except `username`
+   * @param {Robot} robot Hubot instance
+   * @param {string} username Birthday boy
+   * @returns {Void}
+   */
+  const createBirthdayRoom = async (robot, username) => {
+    const message = `Soon our colleague @${username} celebrates his birthday. Let's discuss this`
+    const roomName = `${username}_happy_birthday_room`
+    const users = Object.values(robot.brain.data.users)
+      .filter(user => user.name !== username)
+      .map(user => user.name)
+    console.log('JSON' + JSON.stringify(users))
+    const room = await robot.adapter.api.post('groups.create', {
+      name: roomName,
+      members: users
+    })
+    robot.messageRoom(room.group.name, message)
+  }
+
+  /** 
+   * Check if bot is in the happy birthday group
+   * @param {Robot} robot Hubot instance
+   * @param {string} username birthday boy's username
+   * @returns {boolean}
+   */
+  const isBotInHBGroup = async (robot, username) => {
+    const roomName = `${username}_happy_birthday_room`
```

Comment:
* Let's call the variable `groupName`.
* Let's call the group `${username}-birthday-group`.